Source/Tests/LinuxSyscalls/x32/Thread.cpp

```diff
@@ -68,6 +68,27 @@ namespace FEX::HLE::x32 {
     // Now we need to update the thread's GDT to handle this change
     auto GDT = &Frame->State.gdt[u_info->entry_number];
     GDT->base = u_info->base_addr;
+
+    // With the segment register optimization we need to check all of the segment registers and update.
+#define GETENTRY(x) (x >> 3)
```

Comment:
maybe `#undef` this before the final return, to keep the define localized or perhaps turn it into a lambda like

```cpp
const auto GetEntry = [](auto value) {
  return value >> 3;
};
```

Reply:
Good catch, I originally had it as a define since it was doing more work. Switched over to a lambda.